package/src/main.rs

```diff
@@ -21,6 +21,12 @@ const PATHSEP: &str = ";";
 #[cfg(target_family = "unix")]
 const PATHSEP: &str = ":";
 
+#[cfg(all(target_os = "linux", target_arch = "arm", target_pointer_width = "32"))]
+const ARCH: &str = "armv7l";
```

Comment:
Ok, I know nothing about arm. I tried to give myself a little bit of an education and it seems armv7l, armv7b, armv8l and armv8b are all 32 bit, the v8s being the 32 bit modes of 64bit capable processors and the big-endian variants being rare, but out there. If I have that right, does it make sense to add the big-endian names in examples/run.sh and use aarch32 here much like we otherwise use aarch64?

Reply:
Or, really, is there any reason to buck the trend here and go against `env::consts::ARCH`?

Reply:
See my reply here: https://github.com/a-scie/ptex/pull/253#issuecomment-2534829908

I really think we should stick with `armv7l`. I don't understand how rust can use a `arm` suffix when you set the platform to `armv7-unknown-linux-musleabihf`

I can update the check to also check if it is little endian.

Reply:
Also `armv8l` can run `armv7l` binaries by using `setarch armv7l ./foo.bin`